emba.sh

```diff
@@ -71,6 +73,9 @@ main()
   export MOD_DIR="./modules"
   export VUL_FEED_DB="$EXT_DIR""/allitems.csv"
   export BASE_LINUX_FILES="$CONFIG_DIR""/linux_common_files.txt"
+  export HTML_PATH="./html/html-files"         # directory for html path
```

Comment:
Do we really need this directory? Could we move this to $CONFIG_DIR?

Reply:
best variant would be: `/config/html/style.css`